### PR TITLE
Set prefix as repo address to simplify ESXi boot.cfg design

### PR DESCRIPTION
**Pull request by sunnyqianzhang**
To address Young, Zachary's comment in PR (https://github.com/RackHD/on-http/pull/38), we can set the prefix as repo address in esx-boot-cfg file so that we don't need long "https://........" appear everywhere.
@RackHD/corecommitters @iceiilin @pengz1

### CODE REVIEW

**Comment by rackhdmirror** [comment]
Build finished.

**Comment by sunnyqianzhang** [comment]
This PR need cowork with the following PR to take effective: https://github.com/RackHD/on-tasks/pull/357

**Comment by pengz1** [comment]
+1

File: data/templates/esx-boot-cfg [hunk 1]
```diff
@@ -1,5 +1,6 @@
 bootstate=0
 title=Loading ESXi installer
+prefix=<%=repo%>/
```
Comment (anhou) [inline data/templates/esx-boot-cfg]: doesn't need this '/'?

**Comment by rackhdmirror** [comment]
Build finished.

File: data/templates/esx-boot-cfg (hunk 1 above)
Comment (sunnyqianzhang) [inline data/templates/esx-boot-cfg]: Double confirmed and it works without this '/', thanks @anhou

**Pull request merged**